### PR TITLE
PacReq: grapejuice-git

### LINKED ISSUE #309: PacReq: Grapejuice
Grapejuice is a management tool for playing Roblox using Wine.
Roblox is a fun game played by many children, but it doesn't run using only Wine. Grapejuice is currently available in the AUR, and it works well on arch-based distros. Installing applications from source is not every parent or child's cup of tea, therefore it would be good to simplify the installation process by including it in pacstall-programs.

URL: https://gitlab.com/brinkervii/grapejuice
Building Guide: https://gitlab.com/brinkervii/grapejuice/-/wikis/Installing-from-source/Debian-10-and-similar

### PR DESCRIPTION
**Pull request by Henryws**
Will fix #309

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Fix codacy issues first

**Comment by Henryws** [comment]
> Fix codacy issues first

The AUR follows the exact same method of using a glob

**Review by wizard-28 — CHANGES_REQUESTED** [review]
No body said to remove the glob... Just put `./` in front of Mr. Glob

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+build() {
+  true
+}
+
+install() {
+  export PYTHONPATH="$(pwd)/src"
+  python3 -m grapejuice_packaging linux_package || python -m grapejuice_packaging linux_package
+  cd dist/linux_package
+  sudo mkdir -p /usr/src/pacstall/grapejuice-git/
+  sudo tar -xf ./*.tar.gz -C /usr/src/pacstall/grapejuice-git/
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:24]:
```suggestion
  sudo tar -xf ./*.tar.gz -C /$STOWDIR/$name/
```

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+build() {
+  true
+}
+
+install() {
+  export PYTHONPATH="$(pwd)/src"
+  python3 -m grapejuice_packaging linux_package || python -m grapejuice_packaging linux_package
+  cd dist/linux_package
+  sudo mkdir -p /usr/src/pacstall/grapejuice-git/
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:23]:
```suggestion
  sudo mkdir -p $STOWDIR/$name
```

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  export PYTHONPATH="$(pwd)/src"
+  python3 -m grapejuice_packaging linux_package || python -m grapejuice_packaging linux_package
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:21]: What is this line for?

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `breaks`, `gives`

---

Also does it have icons / desktop files? How is your script handling those?

File: packages/grapejuice-git/grapejuice-git.pacscript (hunk 3 above)
Comment (Henryws) [reply packages/grapejuice-git/grapejuice-git.pacscript:21]: Just in case

**Comment by Henryws** [comment]
> Add `breaks`, `gives`
> 
> Also does it have icons / desktop files? How is your script handling those?

The linux packaging part takes care of icons / desktop files

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,30 @@
+name="grapejuice-git"
+description="A simple wine+roblox management application"
+depends="git python3-pip python3-setuptools python3-wheel python3-dev pkg-config libcairo2-dev gtk-update-icon-cache desktop-file-utils xdg-utils libgirepository1.0-dev gir1.2-gtk-3.0"
+url="https://gitlab.com/brinkervii/grapejuice.git"
+maintainer="TwilightBlood <hwengerstickel@pm.me>"
+gives="grapejuice"
+breaks="grapejuice grapejuice-bin"
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:7]: Add the other types too

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+build() {
+  true
+}
+
+install() {
+  export PYTHONPATH="$(pwd)/src"
+  python3 -m grapejuice_packaging linux_package || python -m grapejuice_packaging linux_package
+  cd dist/linux_package
+  sudo mkdir -p $STOWDIR/$name
+  sudo tar -xf ./*.tar.gz -C /$STOWDIR/$name/
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:27]:
```suggestion
  sudo tar -xf ./*.tar.gz -C "$STOWDIR/$name"
```

File: packages/grapejuice-git/grapejuice-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+build() {
+  true
+}
+
+install() {
+  export PYTHONPATH="$(pwd)/src"
+  python3 -m grapejuice_packaging linux_package || python -m grapejuice_packaging linux_package
+  cd dist/linux_package
+  sudo mkdir -p $STOWDIR/$name
```
Comment (wizard-28) [inline packages/grapejuice-git/grapejuice-git.pacscript:26]:
```suggestion
  sudo mkdir -p "$STOWDIR/$name"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**